pgvectorscale/src/access_method/sbq.rs

```diff
@@ -80,6 +78,66 @@ impl SbqMeans {
     }
 }
 
+#[derive(Archive, Deserialize, Serialize)]
+#[archive(check_bytes)]
+#[repr(C)]
+pub struct SbqMeans {
```

Comment:
I would name this `SbqMeansV2` to avoid confusion when looking at older code that calls something else `SbqMeans`.